src/styles/LoginPage.module.scss

```diff
@@ -1,81 +1,69 @@
 @import "colors";
+@import "_variables";
+@import "breakpoints";
+@import "mixins";
 
 @mixin btnStyle {
-  padding: 0.5rem 2rem;
+  padding: $padding-sm-lg;
   background: $white-color;
-  border: 1px solid $black-color;
-  border-radius: 0.25rem;
+  border: $thin-border;
+  border-radius: $border-radius-xs;
   color: $gray-dark--1;
   font-weight: 600;
   cursor: pointer;
   &:hover {
     background-color: $gray-light--3;
-    box-shadow: 2px 4px 4px $gray-light--1;
+    box-shadow: $box-shadow-lg;
   }
 }
 
 .login-page {
   &__container {
-    height: 21.875rem;
+    height: 23.75rem;
     width: 18.75rem;
-    padding: 0.5rem;
+    padding: $padding-xs;
     background: $white-color;
     display: flex;
     flex-direction: column;
     align-items: center;
-    border: 2px solid #000;
-    border-radius: 0.25rem;
+    border: $thick-border--dark;
+    border-radius: $border-radius-xs;
     position: absolute;
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
-  }
+    box-shadow: $box-shadow-xl;
 
-  &__btn {
-    padding: 0.3rem 2rem;
-    background: $gray-light--2;
-    border: 1px solid $black-color;
-    border-radius: 0.25rem;
-    color: $gray-dark--1;
-    font-weight: 600;
-    cursor: pointer;
-    &:hover {
-      background-color: $gray-dark--1;
-      color: $white-color;
-      box-shadow: 2px 4px 4px $gray-light--1;
-    }
-    &.inactive {
-      background: $white-color;
-      color: $gray-dark--1;
-      border: 1px solid $gray-dark--1;
-      cursor: default;
-      border: none;
-      box-shadow: none;
+    @include breakpoint-sm-and-lower {
+      width: 15rem;
     }
   }
 }
 
-.button-container {
-  position: absolute;
-  top: 0;
-  left: 25px;
-  width: 80%;
-  transition: left 0.5s ease;
-  border: 1px solid $gray-dark--1;
-  border-radius: 4px;
-  padding: 4px;
-  margin-bottom: 0.5rem;
-}
+.signup {
+  &__container {
+    margin-top: 1.5rem;
```

Comment:
go over css and use spacing variables

Reply:
and to be honest, it is a nice way to reuse the base class name with &, but there is a big downside to this. Visibility and readability. 

if the scss file is big, you can get lost for not knowing where are you in the file, just seeing modifier and not base. second, when you would like to look at class name from jsx, you copy it and search it, right? but with this, you can't do that. because lets say we have a `.singnup__container--white` no way you would find it from the get go. but you would have to search `.singnup` and then manually find container etc... 
so let's unravel this